uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/HRContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -10,23 +10,23 @@
             <button class="btn btn-primary-outline">
```

Comment:
Do we still need to keep the static template?

Reply:
Honestly I don't know, I confirmed with @bmnepali that he still needs it.

Reply:
What does he need it for?

Reply:
He said he made some changes to the files in his current PR, so after that gets merged, they can safely be deleted.